sys-block/lsiutil/lsiutil-1.62-r3.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="LSI Logic Fusion MPT Command Line Interface management tool"
+HOMEPAGE="https://www.broadcom.com/"
+SRC_URI="https://docs.broadcom.com/docs-and-downloads/legacy-host-bus-adapters/legacy-host-bus-adapters-common-files/LSIUtil_${PV/./-}.zip"
+
+# This package can never enter stable, it can't be mirrored and upstream
+# can remove the distfiles from their mirror anytime.
+LICENSE="LSI"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~x86"
+RESTRICT="mirror bindist"
+
+DEPEND="app-arch/unzip"
```

Comment:
So you migrated a package to EAPI 7 which is cool -- thank you. But you forgot about what EAPI 7 is about ;-)
Please read about `BDEPEND`.

Reply:
Thanks, fixed.